### ISSUE TITLE
Oxlint has a panic in DefinitelyTyped

### ISSUE THREAD

**Issue by Dunqing** [oxc-project/oxc#2087]
<img width="1470" alt="image" src="https://github.com/oxc-project/oxc/assets/29533304/adfaa890-de3c-4eca-ac00-e975aea760ab">

thread `<unnamed>` panicked at [crates/oxc_parser/src/lexer/mod.rs:324:40](https://github.com/oxc-project/oxc/blob/45dffcc6ac960dc2323c19658f9c892cdf214942/crates/oxc_parser/src/lexer/mod.rs#L324):

The issue was caused by PR #2010.

**Comment by Boshen** [comment]
Note to self: add to ecosystem CI

**Comment by Boshen** [comment]
Found it:
```
interface Helpers {
  inspect(): `~~~~\n${string}\n~~~~`;
}
```

**Comment by Boshen** [comment]
Fixed in v0.2.2.

Ecosystem CI was never ever triggered  ... fixing it now.
